experimental/dds/tree2/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -274,16 +274,7 @@ const nodeProxyHandler: AdaptingProxyHandler<NodeProxyTarget, EditableTree> = {
 		);
 		if (typeof key === "string" || symbolIsFieldKey(key)) {
 			const fieldKey: FieldKey = brand(key);
-			if (fieldKey === localNodeKeySymbol) {
```

Comment:
Yea, I guess we don't really need this in the near term (or perhaps ever). Thanks.

Reply:
this code was a bug: localNodeKeySymbol should not have been a fieldKey symbol. Its just a regular API on editable tree nodes (like type or parent) , not a field.